_posts/2015-12-09-foreman-community-newsletter-december-2015.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2015
+author: Tomer Brisker
+tags:
+- foreman
+- katello
+- newsletter
+- openscap
+- xen
+- remote execution
+- high availability
+- windows imaging
+excerpt: |
+  Foreman 1.10 and Katello 2.4 coming soon
+---
+
+### Foreman 1.10 RC3 is out, 1.10 expected soon
+
+Several regressions that were discovered in the release candidates have delayed the final release of 1.10.
+In an attempt to deliver a stable version, we have decided to postpone the release until those issues are resolved.
+If all goes well, the final 1.10 version should be released within a few days.
+Thank you to all involved in testing and fixing the RCs, this helps us maintain a high standard for our releases!
+
+### Katello 2.4 RC3 is out, 2.4 expected soon
+
+Katello 2.4 is planned to be released in alignment with Foreman 1.10.
+This release will primarily provide compatibility with Foreman 1.10, convert most objects from Elasticsearch to scoped search and fix a number of bugs.
+Check out the [release notes](http://www.katello.org/docs/2.4/release_notes/release_notes.html) for a full list of updates.
+Please help us with testing this RC so the final release will be as solid as possible!
+
+### New and updated plugins this month
+
+-   [foreman\_openscap](https://github.com/theforeman/foreman_openscap) and
+    [smart\_proxy\_openscap](https://github.com/theforeman/smart_proxy_openscap)
+    0.5.0 was released Novemeber 1st. Please note, this version is only compatible with Foreman 1.11 (current develop branch).
+-   [foreman\_remote\_execution](https://github.com/theforeman/foreman_remote_execution)
+    0.1.1 was released November 12th.
+-   [foreman-xen](https://github.com/theforeman/foreman-xen)
+    0.2.3 (for Foreman >= 1.10) and 0.1.7 (for older versions) were released Novemeber 17th.
+-   [foreman-tasks](https://github.com/theforeman/foreman-tasks)
+    0.7.7 was released December 1st.
+
+### Other Updates
+
+-   A new [feature](https://groups.google.com/forum/#!topic/foreman-users/3RYfo0S5c8M) is planned to provide more information about smart proxies in Foreman, we would be happy to hear your input on this matter.
+-   An initial [Developer Handbook](http://theforeman.org/handbook.html) has been created to ease the entry of new contributers and help maintain high standards of existing ones.
+    Feel free to fork and open a pull request if you feel some important information is missing!
+-   The Foreman installer has been updated to drop puppet 2.7 support starting with the next version of Foreman (1.11).
+    For more information please read the [announcement](https://groups.google.com/forum/#!topic/foreman-users/YQNfxUzG_As).
+-   The development infrastructure has seen some updates - Redmine has been updated to version 2.6.8 and Jenkins has been updated to version 1.625.2.
```

Comment:
A mention of the review-culture plan thread and/or blog post would be good, I think.